shared/js/background/utils.es6.js

```diff
@@ -17,6 +18,29 @@ function extractHostFromURL (url, shouldKeepWWW) {
     return hostname
 }
 
+// Removes information from a URL, such as path, user information, and optionally sub domains
+function extractLimitedDomainFromURL (url, keepSubdomains) {
+    if (!url) return ''
```

Comment:
In most cases we return undefined but here we are returning empty string - is there a difference? should we use the same thing for consistency?